setup.cfg

```diff
@@ -6,3 +6,6 @@ python_files=test_*.py
 
 [flake8]
 max-line-length=79
+ignore=I002
+exclude=
+  doc/
```

Comment:
Can you add this configuration to the `.stickler.yml` file, too? That will help keep things in sync.

See https://stickler-ci.com/docs#python

Reply:
Good idea, done 
(though ideally it should read from standard configs...)

Reply:
It looks like this didn't actually work to ignore the doc directories -- see all the Stickler messages in https://github.com/pydata/xarray/pull/1930